ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -574,15 +631,16 @@ export function PreviewConcierge(props: any) {
             parentModelId ? models[parentModelId].craftercms.path : null
           ).subscribe(
             () => {
+              guestToHost$.next({ type: FETCH_GUEST_MODEL });
```

Comment:
Don’t like the idea of using this channel for this purpose.
Let’s find a better way of reusing the loading logic.